Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -0,0 +1,833 @@
+namespace gdjs {
+  declare var cordova: any;
+
+  const logger = new gdjs.Logger('Player Authentication');
+  export namespace playerAuthentication {
+    // Authentication information.
+    let _username: string | null = null;
+    let _userId: string | null = null;
+    let _userToken: string | null = null;
+    let _justLoggedIn = false;
+
+    let _error: any = null;
+    let _checkedLocalStorage: boolean = false;
+
+    // Authentication display
```

Comment:
Do you think it would be easy to spit it in 2 files: one for the logic and one for the presentation?

Reply:
That is a good idea, I'll see if I can split that